docs/design/coreclr/profiling/davbr-blog-archive/Profiler stack walking Basics and beyond.md

```diff
@@ -145,77 +145,77 @@ But before you get too deep, note that the issue of whether and how to seed a st
 
 For the truly adventurous profiler that is doing an asynchronous, cross-thread, seeded stack walk while filling in the unmanaged holes, here’s what it would look like.
 
-| 
+|
 
-Block of  
-Unmanaged  
+Block of
+Unmanaged
 Frames
 
- | 
-1. You suspend the target thread (target thread’s suspend count is now 1) 
-2. You get the target thread’s current register context 
-3. You determine if the register context points to unmanaged code (e.g., call ICorProfilerInfo2::GetFunctionFromIP(), and see if you get back a 0 FunctionID) 
+ |
+1. You suspend the target thread (target thread’s suspend count is now 1)
+2. You get the target thread’s current register context
+3. You determine if the register context points to unmanaged code (e.g., call ICorProfilerInfo2::GetFunctionFromIP(), and see if you get back a 0 FunctionID)
 4. In this case the register context does point to unmanaged code, so you perform an unmanaged stack walk until you find the top-most managed frame (D)
  |
-| 
+|
 
-Function D  
+Function D
 (Managed)
 
- | 
+ |
 1. You call DoStackSnapshot with your seed context. CLR suspends target thread again: its suspend count is now 2.  Our sandwich begins.
 
 1. CLR calls your StackSnapshotCallback with FunctionID for D.
  |
-| 
+|
```

Comment:
These again appear on the same line before and after the change. I'm not sure about the desired output.